tests/tests_common.py

```diff
@@ -289,53 +288,29 @@ def test_subscribe_events_to_wrong_node(self):
             handle = sub.subscribe_events(v)
         sub.delete()
 
-    def test_events_deprecated(self):
-        msclt = MySubHandlerDeprecated()
-        sub = self.opc.create_subscription(100, msclt)
-        handle = sub.subscribe_events()
-
-        ev = Event(self.srv.iserver.isession)
-        msg = b"this is my msg "
-        ev.Message.Text = msg
-        tid = datetime.utcnow()
-        ev.Time = tid
-        ev.Severity = 500
-        ev.trigger()
-
-        clthandle, ev = msclt.future.result()
-        self.assertIsNot(ev, None)  # we did not receive event
-        self.assertEqual(ev.Message.Text, msg)
-        #self.assertEqual(msclt.ev.Time, tid)
-        self.assertEqual(ev.Severity, 500)
-        self.assertEqual(ev.SourceNode, self.opc.get_server_node().nodeid)
-
-        # time.sleep(0.1)
-        sub.unsubscribe(handle)
-        sub.delete()
-
-    def test_events(self):
-        msclt = MySubHandler()
-        sub = self.opc.create_subscription(100, msclt)
-        handle = sub.subscribe_events()
-
-        ev = Event(self.srv.iserver.isession)
-        msg = b"this is my msg "
-        ev.Message.Text = msg
-        tid = datetime.utcnow()
-        ev.Time = tid
-        ev.Severity = 500
-        ev.trigger()
-
-        ev = msclt.future.result()
-        self.assertIsNot(ev, None)  # we did not receive event
-        self.assertEqual(ev.Message.Text, msg)
-        #self.assertEqual(msclt.ev.Time, tid)
-        self.assertEqual(ev.Severity, 500)
-        self.assertEqual(ev.SourceNode, self.opc.get_server_node().nodeid)
-
-        # time.sleep(0.1)
-        sub.unsubscribe(handle)
-        sub.delete()
+    #def test_events(self):
+        #msclt = MySubHandler()
+        #sub = self.opc.create_subscription(100, msclt)
+        #handle = sub.subscribe_events()
+
+        #ev = EventGenerator(self.srv.iserver.isession)
+        #msg = b"this is my msg "
+        #ev.Message.Text = msg
+        #tid = datetime.utcnow()
+        #ev.Time = tid
+        #ev.Severity = 500
+        #ev.trigger()
+
+        #ev = msclt.future.result()
+        #self.assertIsNot(ev, None)  # we did not receive event
+        #self.assertEqual(ev.SourceNode, self.opc.get_server_node().nodeid)
+        #self.assertEqual(ev.Message.Text, msg)
+        ##self.assertEqual(msclt.ev.Time, tid)
+        #self.assertEqual(ev.Severity, 500)
+
+        ## time.sleep(0.1)
+        #sub.unsubscribe(handle)
+        #sub.delete()
```

Comment:
Yes I will do it. I am currently adding more tests and debugging. This was just one intermediate commit. My computer was strange today, so I push it as back up, and you can see the progress :)